instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/internal/BedrockRuntimeAttributesGetter.java

```diff
@@ -117,11 +118,11 @@ public Double getRequestTopP(ExecutionAttributes executionAttributes) {
   @Override
   public List<String> getResponseFinishReasons(
       ExecutionAttributes executionAttributes, Response response) {
-    String stopReason = BedrockRuntimeAccess.getStopReason(response.getSdkResponse());
-    if (stopReason == null) {
+    List<String> stopReasons = BedrockRuntimeAccess.getStopReasons(response);
+    if (stopReasons == null) {
       return null;
```

Comment:
I think that we currently return empty list instead of `null` from other getters

Reply:
Thanks - I think we can't do that for the request stop sequences to differentiate from user setting to empty vs not setting, but for response here empty seems good.

Reply:
Thanks, removed `@Nullable` and had the extractor check for empty. I left the Getter permitting `null` anyways since I couldn't see a strong reason to be too strict on it (I noticed `MessagingAttributesGetter.getMesssageHeader` being strict though)

Reply:
I'll leave this to @trask to decide whether it makes sense to keep the `@Nullable` on the getter.

Reply:
I'm ok with @Nullable List as long as there's a comment in the code explaining what the difference between null and empty signals to the caller since it's never obvious to me

Reply:
Thanks - actually no comment comes to mind so I removed the `Nullable` but kept the null check to avoid crashing if instrumentation has a bug, realized that's probably closer to the practice of what's in the OTel APIs.